Onboarding Documents/Communication and Transparency.md

```diff
@@ -0,0 +1,76 @@
+# Communication and Transparency
+
+The best way for you to make a decision about what’s good for Clef is to have all of the information about what’s going on at Clef. The same is true for everyone else you work with, so it’s important that we’re all up to date about what’s happening, even with the areas of Clef that we’re not actively involved with.
+
+Transparency may seem like a passive process, like "don’t hide anything," but it’s actually a big challenge to keep everyone up to date and on the same page about stuff they’re not working on on a daily basis, even for a company the size of Clef. These are some of the policies we use to make information more accessible, but our work is ongoing.
+
+## Communication
+
+
+### Morning Meetings
```

Comment:
I'm not sure that morning meetings and friday updates should be part of our policy -- if we want to change those, I don't think we want to change policy